modules/tensor_mechanics/doc/content/source/materials/NonLocalDamage.md

```diff
@@ -0,0 +1,28 @@
+# Scalar Material Damage
+
+!syntax description /Materials/NonLocalDamage
+
+## Description
+
+`NonLocalDamage` is a model to define the effect of damage on the stress and stiffness in a continuum damage mechanics setting. It does not directly compute the stress, but must be used in conjunction with [ComputeDamageStress](/ComputeDamageStress.md).
+
+This model is a scalar damage model in which the stress $\boldsymbol{\sigma}$ is computed as a function of the damage $d$, the original stiffness of the material $\mathbb{C}$ and the elastic strain $\boldsymbol{\varepsilon}$:
```

Comment:
I don't think there's any need to include the equation showing how the damage index is used, since that is included in the documentation for other parts of the damage system. In this documentation, I'd mainly say how this is used as an adapter to get the nonlocal damage from the RadialAverage object and provide it for use with the stress calculation. Make sure you use a link to RadialAverage (like you've already done for ComputeDamageStress above).